libs/langchain-community/langchain.config.js

```diff
@@ -91,6 +91,7 @@ export const config = {
     // llms
     "llms/ai21": "llms/ai21",
     "llms/aleph_alpha": "llms/aleph_alpha",
+    "llms/arcjet": "llms/arcjet",
```

Comment:
Will need to add these under `requiresOptionalDependency` below but I can do that